x-pack/packages/kbn-cloud-security-posture/graph/src/components/node/diamond_node.tsx

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { memo } from 'react';
+import { useEuiBackgroundColor, useEuiTheme } from '@elastic/eui';
+import { Handle, Position } from '@xyflow/react';
+import type { EntityNodeViewModel, NodeProps } from '../types';
+import {
+  NodeContainer,
+  NodeLabel,
+  NodeShapeOnHoverSvg,
+  NodeShapeSvg,
+  NodeIcon,
+  NodeButton,
+  HandleStyleOverride,
+} from './styles';
+
+export const DiamondNode: React.FC<NodeProps> = memo((props: NodeProps) => {
+  const { id, color, icon, label, interactive, expandButtonClick } =
+    props.data as EntityNodeViewModel;
+  const { euiTheme } = useEuiTheme();
+  return (
+    <NodeContainer>
+      {!interactive || (
+        <NodeShapeOnHoverSvg
+          width="99"
+          height="98"
```

Comment:
could we add those values to a constant and reuse them on viewBox and NodeButton's x and y calculation below?